src/shared/network.ts

```diff
@@ -1,12 +1,30 @@
 import { Networking } from "@flamework/networking";
 
-interface ServerEvents {}
+/*
 
-interface ClientEvents {}
+	Server Recievers
+	Network messages that can be sent from clients and recieved from the server
+
+*/
+interface ServerEvents {}
 
 interface ServerFunctions {}
 
+/*
+
+	Client Recievers
+	Network messages that will be sent from the server and recieved by client(s)
+
+*/
+interface ClientEvents {}
+
 interface ClientFunctions {}
```

Comment:
I'm not a fan of the section comments here and I think it'd make more sense to use doc comments on each individual interface.

I also think a simpler message like `Client to Server events` or `Server to Client functions` would be better for those.